font_js.go

```diff
@@ -0,0 +1,106 @@
+// +build netgo
+
+package engo
+
+//TODO go generate to convert fonts to ones readable by engo
```

Comment:
That's pretty much what we do already. So "copying" that to gopherjs wouldn't be that bad.